src/pages/ProductPage.module.css

```diff
@@ -0,0 +1,35 @@
+.MainProductPage {
```

Comment:
Why not .ProductPage?

Reply:
Maybe just to indicate that it is the main product page :)